src/EA.Weee.Web/Views/OrganisationRegistration/RegistrationComplete.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@using EA.Weee.Web.RazorHelpers
+@{
+    const string title = "Organisation created";
+    ViewBag.Title = title + " - GOV.UK";
+}
+@section full_main_content
+{
+    @using (Html.BeginForm())
+    {
+        @Html.AntiForgeryToken()
+        @Html.Gds().ValidationSummary()
+
+        <div class="govuk-panel govuk-panel--confirmation">
+            <h1 class="govuk-panel__title">
+                @title
+            </h1>
+        </div>
+
+        <p>Your organisation has been created.</p>
+        <h3 class="govuk-heading-m">What happens next</h3>
```

Comment:
@stevelee-civica  Sorry missed this, change this to h2 as Wave will moan about heading heirachy